3 reports.sql

```diff
@@ -0,0 +1,31 @@
+use ScenicHotelDB 
+go 
+
+/*
+1) Show me a list of all the sceniors in the scenior section in this format: 
+Firstname lastname, age, season(Summer is from May to October, The rest is winter), and  amount of days spent in the hotel. (ex:Jimmy Smith, 76, winter, 5).
+*/
+
+select SeniorInfo = concat(g.FirstName, ' ', g.LastName, ', ', datediff(year, DateOfBirth, DateCheckedIn), ', ', g.Season, ', ', g.AmountOfDaysSpentInHotel ) 
+from guest g
+where g.RoomType = 'Senior Room'
+
+--2) Show me The average days that people stay in the hotel.
+
+select AvgDaysSpentInHotel = avg(AmountOfDaysSpentInHotel) 
+from guest g
+
+--3) I would like to have a list of those who visited in the summer, stayed for over seven days and were 2 or more people.
+
+select * 
+from guest g 
+where g.Season = 'Summer'
+and AmountOfDaysSpentInHotel > 7 
+and AmountOfPeople > 2
+
+--4) Show me a list of those Guest who came back more than once.
+
```

Comment:
_This one was a little tricky as only those who came back twice are supposed to be listed not those who took more than one room._

Reply:
I corrected this as well

Reply:
I Think your select statement deos not show the right results it is quite complicating but **you can do it:)**

Reply:
Guess what? I figured it out! Hope it's correct this time around:)
Pls. let me know.........

Reply:
**Yup you did it the right way this time. amazing!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!**